requirements.txt

```diff
@@ -10,6 +10,7 @@ async-lru==2.0.4
 attrs==23.1.0
 Babel==2.13.1
 backcall==0.2.0
+bcrypt==4.1.2
```

Comment:
also why?

Reply:
I save the encrypted key as a password, therefore, even when the key is leaked (not likely), it is less likely to exploit our service